src/xdocs/checks/sizes/parameternumber.xml

```diff
@@ -112,6 +119,60 @@
 public void needsLotsOfParameters(int a,
     int b, int c, int d, int e, int f, int g, int h) {
     ...
+}
+        </source>
+        <p id="Example4-config">
+          To configure the check to ignore methods and constructors annotated with
+          <code>JsonCreator</code> annotation:
+        </p>
+        <source>
+&lt;module name=&quot;Checker&quot;&gt;
+  &lt;module name=&quot;TreeWalker&quot;&gt;
+    &lt;module name=&quot;ParameterNumber&quot;&gt;
+      &lt;property name=&quot;ignoreAnnotatedBy&quot; value=&quot;JsonCreator&quot;/&gt;
+    &lt;/module&gt;
+  &lt;/module&gt;
+&lt;/module&gt;
+        </source>
+        <p id="Example4-code"> Example:</p>
+        <source>
+class Example {
+    @JsonCreator
+    Example(int a, int b, int c, int d,
+            int e, int f, int g, int h) {}
+
+    @interface JsonCreator {}
+}
+        </source>
+        <p>
+          Fully qualified class name(s) can also specified in ignoreAnnotatedBy property.
+          Note that when specifying annotation(s), only methods and constructors annotated with
+          the exact specified annotation(s) will be ignored, any variation, even if it refers to
+          the same annotation, will result in it being checked, and produce violation if any.
+        </p>
```

Comment:
Let's be as concise as possible:
```suggestion
        <p>
          Note: annotation names specified in the <code>ignoreAnnotatedBy</code> property must be an exact match for the annotation name on the method or constructor.
        </p>
```

This statement + the extension to the example I've provided above should sufficiently convey this concept.

Reply:
Done.